playkit/src/main/java/com/kaltura/playkit/PKMediaEntryInterceptor.java

```diff
@@ -0,0 +1,8 @@
+package com.kaltura.playkit;
+
+/**
+ * Created by alex_lytvynenko on 09.11.2020.
+ */
+public interface PKMediaEntryInterceptor {
+    void apply(PKMediaEntry mediaEntry, OnMediaInterceptorListener listener);
+}
```

Comment:
This class has to move to Kaltura Player.

Reply:
If we move it to KalturaPlayer, Broadpeak plugin will have to add kaltura-player dependency as well. Are you sure?

Reply:
See above - it just doesn't belong to PlayKit.

Reply:
ok, i will move it to KalturaPlayer